src/components/books.js

```diff
@@ -1,25 +1,26 @@
-import React, { useState } from 'react';
-import Book from './book';
-import AddBook from './AddBook';
+/* eslint-disable import/extensions */
+/* eslint-disable react/prop-types */
+import React from 'react';
+import Book from './Book';
+import AddBook from './addBook';
```

Comment:
Similar to the comment made above, you have imported `./addBook` while the saved file is ./AddBook`.

Kindly modify them to remain consistent so that your project can run without errors on all devices.